crates/core_arch/examples/neon_gen.rs

```diff
@@ -0,0 +1,502 @@
+use std::fs::{self, File};
```

Comment:
An example does not feel like the right place to put this generator. It might be better as its own crate within the workspace (e.g. `stdarch-neon-gen` or similar maybe?).

Reply:
I remove the generator and spec, this should solve it.

Reply:
The generator should also have a test that runs the generator in "check mode" and panics on diffs, such that `cargo test --all` in CI executes it. Probably also worth checking that the generator actually runs on CI and rejects manually modified `generate.rs`s.

Reply:
Moved to https://github.com/rust-lang/stdarch/pull/802